rest/data/task.go

```diff
@@ -228,7 +133,7 @@ func (tc *DBTaskConnector) CheckTaskSecret(taskID string, r *http.Request) (int,
 }
 
 // GetManifestByTask finds the manifest corresponding to the given task.
-func (tc *DBTaskConnector) GetManifestByTask(taskId string) (*manifest.Manifest, error) {
+func GetManifestByTask(taskId string) (*manifest.Manifest, error) {
```

Comment:
Is it possible to move this into model/manifest? Or move the logic into the getManifestHandler directly.

Reply:
Yup, that fixes the import cycle error I found when moving it into model/task